Cargo.toml

```diff
@@ -14,7 +14,7 @@ doctest = false
 
 [dependencies]
 tokio-postgres = "0.5.4"
-tokio-pg-mapper-derive = { version = "0.1.5", optional = true }
+tokio-pg-mapper-derive = { path = "pg_mapper_derive", optional = true }
```

Comment:
I'm intentionally not local-importing the derive crate.  Consequently, this change should be removed.

Reply:
@Dowwie can you explain why?

Reply:
These are separately released crates on crates.io.  Using versioning ensures no mistakes are made.